pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPDConfiguration.java

```diff
@@ -382,11 +413,22 @@ public void setRenderer(Renderer renderer) {
         this.cpdRenderer = null;
     }
 
+    /**
+     * @deprecated Use {@link #setRenderer(CPDReportRenderer)} instead
+     * @param renderer
+     */
+    @Deprecated
     public void setCPDRenderer(CPDRenderer renderer) {
         this.cpdRenderer = renderer;
         this.renderer = null;
     }
 
+    public void setRenderer(CPDReportRenderer renderer) {
```

Comment:
I think, we should make this package-private for now as well. The only way to set a renderer is by name.